com.github.paulwedeck.settlers-remake.metainfo.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppdataEdit 5.1-->
+  <id>com.github.paulwedeck.settlers-remake</id>
```

Comment:
It should be `io.github` as it's "user-generated content" and not an app from GitHub.